policybot/dashboard/topics/releasequalification/topic.go

```diff
@@ -38,22 +38,34 @@ type ReleaseQualification struct {
 
 // SingleMonitorStatus represents the status of one single monitor
 type SingleMonitorStatus struct {
-	Name        string
-	Status      string
-	TestID      string
-	ProjectID   string
-	ClusterName string
-	FiredTimes  int64
-	UpdatedTime time.Time
+	Name          string
+	Status        string
+	TestID        string
+	Branch        string
+	Description   string
+	FiredTimes    int64
+	LastFiredTime string
```

Comment:
Why are we storing times as int64 and string?

Reply:
FiredTimes is actually a counter of how many times the alarm fired so should be int64
The lastFiredTime is defined as TIMESTAMP in spanner, here use string because we may need to possibly reformat the output when render on the page